Tools/TestWebKitAPI/Tests/WebKitCocoa/Navigation.mm

```diff
@@ -1798,3 +1798,60 @@ HTTPServer server({
     while (navigationToExample)
         Util::spinRunLoop();
 }
+
+TEST(WKNavigation, Multiple303Redirects)
```

Comment:
Note that in general, we prefer layout tests to API tests whenever possible. This is a case where a layout test was possible I believe.